primitives/runtime-interface/src/impls.rs

```diff
@@ -497,7 +497,7 @@ macro_rules! for_u128_i128 {
 			type Owned = ();
 
 			fn into_ffi_value(&self) -> WrappedFFIValue<u32> {
-				unsafe { (mem::transmute::<&Self, *const u8>(self) as u32).into() }
```

Comment:
unsafe not required according to clippy here. It was suggesting `*self as *const u128 as *const u8 as u32` but I think the above is equivilent.

Reply:
It's got specific tests around it and they pass, so I'm happy on this one.